Mobile devices

deadlight opened the pull request:
## Done

New mobile devices page and associated styling

## QA

Check `/mobile/devices` in all viewports and check it against the [copydoc](https://docs.google.com/document/d/1l_P2yEVeF6tIvSsm7LAO9ds9nf_ZenKhwlOe3xZybBc/edit#)

nottrobin commented on templates/mobile/shared/_devices_newsletter.html:
```diff
@@ -0,0 +1,40 @@
+<div id="mc_embed_signup" class="box-digest promo-signup">
```
I assume other things is gonna use this include?

deadlight replied:
yes

nottrobin commented:
## Differences from copy docs

Please update either markup or copy doc:

- "BQ Aquaris M10 Ubuntu Edition" - copy doc has "tablet" on the end
- Copy code has no description under this ^ heading
- s/Micro HDMI port/Micro HDMI slot/
- "Alternatively, flash your own device" is a H2 in copy doc, H3 in page
- Under this heading ^ copy doc says "Find out more >", markup says "Find out how to install Ubuntu on a mobile device" (Notice missing chevron as well as different text)
- Under "Did you know Ubuntu is available on IoT devices too", missing link to Ubuntu Core

## Newsletter sign-up

- No error message on error, just red box around element. Is that expected?
- Confirmation message looks unstyled:

![confirmation message](https://assets.ubuntu.com/v1/0c02f36e-confirm-message.png)
![small screen](https://assets.ubuntu.com/v1/e236bc83-small-screen-confirm-message.png)

nottrobin commented on templates/mobile/devices.html:
```diff
@@ -4,19 +4,182 @@
+            <h1>Mobile devices</h1>
+            <p>The Ubuntu experience is available on a number of commercial devices. </p>
+            <ul class="list-ubuntu">
+                <li>Mobile devices that  turn into PCs*</li>
```
Shouldn't the `*` here be superscript or something?

deadlight replied:
We explored options and decided on normal text styling.

deadlight commented:
- "BQ Aquaris M10 Ubuntu Edition" - copy doc has "tablet" on the end - **fixed**
 - Copy code has no description under this ^ heading - **removed**
 - s/Micro HDMI port/Micro HDMI slot/ - **fixed**
 - "Alternatively, flash your own device" is a H2 in copy doc, H3 in page - **this is intentional, copydoc doesn't reflect the structure perfectly but I've changed the copydoc to avoid confusion**
 - Under this heading ^ copy doc says "Find out more >", markup says "Find out how to install Ubuntu on a mobile device" (Notice missing chevron as well as different text) - **I updated the copydoc and the chevron was erroneous (external links never have one)**
 - Under "Did you know Ubuntu is available on IoT devices too", missing link to Ubuntu Core - **done**

Newsletter fixes to come.

deadlight commented:
@nottrobin Newsletter form feedback cleaned up. I'll put in an issue to look at how we style those, globally.
